externs/shaka/manifest_parser.js

```diff
@@ -96,12 +96,20 @@ shaka.extern.ManifestParser = class {
   onExpirationUpdated(sessionId, expiration) {}
 
   /**
-   * Tell the parser that the initial variant has been chosen.
+   * Tells the parser that the initial variant has been chosen.
    *
    * @param {shaka.extern.Variant} variant
    * @exportDoc
    */
   onInitialVariantChosen(variant) {}
+
+  /**
+   * Tells the parser that a location should be banned.
```

Comment:
You should specify this is called on retry.

Reply:
Done!